com.microsoft.Edge.yaml

```diff
@@ -0,0 +1,155 @@
+  - '--socket=wayland'
+  - '--require-version=1.8.2'
+  - '--system-talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--system-talk-name=org.freedesktop.Avahi'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+#  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+#  - '--filesystem=xdg-music'
+#  - '--filesystem=xdg-videos'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: libcups
+    make-args: [libs]
+    no-make-install: true
+    post-install:
+      - make install-headers install-libs
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/apple/cups/releases/download/v2.3.3/cups-2.3.3-source.tar.gz
+        sha256: 261fd948bce8647b6d5cb2a1784f0c24cc52b5c4e827b71d726020bcc502f3ee
+
+  - name: gtk-cups-backend
+    buildsystem: meson
+    make-args: [modules/printbackends/libprintbackend-cups.so]
+    no-make-install: true
+    post-install:
+      - install -Dm 755 modules/printbackends/libprintbackend-cups.so -t /app/lib/gtkmodules/3.0.0/printbackends/
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gtk.git
+        tag: 3.24.23
+
+  - name: gtk-settings
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 644 gtk-settings.ini /app/etc/xdg/gtk-3.0/settings.ini'
+    sources:
+      - type: file
+        path: gtk-settings.ini
+
+  - name: pipewire
+    buildsystem: meson
+    config-opts:
+      - -Dgstreamer=disabled
+      - -Dman=false
+      - -Dsystemd=false
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/pipewire/pipewire.git
+        tag: 0.2.7
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        # branch: master
+        # commit: f419c2e36d9f6f6115798d946e3630b64d3bd075
+        tag: v2020.10beta.4
+      # - type: dir
+      #   path: ../../code/zypak
+      - type: patch
+        paths:
+          - patches/01-msedge-sandbox.patch
+
+  - name: flextop
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/refi64/flextop
+        commit: 485c332c7ec36a8b74be8f3622fc756ac6263f40
+
+  - name: edge
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 edge.sh /app/bin/edge'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 stub_sandbox.sh /app/bin/stub_sandbox'
+      - 'install -Dm 644 -t /app/share/flatpak-edge flatpak_policy.json'
+      - 'install -Dm 644 -t /app/share/flatpak-edge mimic_warning.txt'
+      - 'install -Dm 644 -t /app/share/applications com.microsoft.Edge.desktop'
+      - 'install -Dm 644 -t /app/share/metainfo com.microsoft.Edge.metainfo.xml'
+      - |
+          for icon in 16 24 32 48 64 128 256; do
```

Comment:
256 is enough.

Reply:
https://github.com/flathub/flathub/pull/1992/commits/94d76fb37f2892a2d2acb81c4bf702e9e5248a0b